index.js

```diff
@@ -2,15 +2,17 @@
 var spliceString = require('splice-string');
 
 module.exports = function (str, cols) {
+	var pre = '';
 	var ret = '';
 	var insideEscape = false;
+	var endCode = '\u001b[39m';
```

Comment:
Shouldn't it support other end-codes like `49`? https://github.com/chalk/ansi-styles/blob/master/index.js